packages/psss-server/src/routes/alerts/ArchiveAlertRoute.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+ import { RespondingError } from '@tupaia/utils';
+import { Route } from '../Route';
+
+export class ArchiveAlertRoute extends Route {
+  async buildResponse() {
+    const { alertId } = this.req.params;
+    const alertSurveyResponse = await this.meditrakConnection?.findSurveyResponseById(alertId);
+
+    if (!alertSurveyResponse) {
+      throw new RespondingError('Alert cannot be found', 500);
+    }
```

Comment:
We can also check if the alert is already archived by fetching the Active or Archived Alerts report. But not sure if that's super necessary